.travis.yml

```diff
@@ -8,6 +8,15 @@ jobs:
     jdk: openjdk8
   - os: linux
     jdk: openjdk11
+  - build: ppc64le
+    os: linux
+    arch: ppc64le
+    jdk: openjdk11
+    script: mvn install -DskipTests
```

Comment:
Tests are failing on ppc or why are skipped?

Reply:
The dependent docker-images ( for these tests ) are supplied via strimzi-kafka-operator, we are working with community to get them enabled/ published for ppc64le architecture, currently as a work around, we are skipping the tests.

Reply:
I think that ideally the tests needing the other Docker Images should be marked as Integration Tests and we should use just `skipITs`. The Unit Tests should not need anything and should be able to pass even without it. 

If the ITs are not marked as ITs today, can you please open a separate issue for it?

Reply:
I am afraid to even the unit tests use that image for Kafka. TBH I am not sure how to easily solve it. We can refactor the code and remove usage of it or build Kafka image for ppc...not sure which option is worse :/

Reply:
Well, as I said ... that is wrong. Unit tests should not depend on any other components such as Kafka. So IMHO anything depending on a Kafka container is IT and should be marked as such.

Reply:
Makes sense. @krishvoor issue for that would be nice :) thanks

Reply:
Ok, I opened https://github.com/strimzi/strimzi-kafka-bridge/issues/508 for this.